src/CloudNative.CloudEvents/CloudNative.CloudEvents.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <PackageVersion>1.0</PackageVersion>
     <Description>CNCF CloudEvents SDK</Description>
+    <LangVersion>8.0</LangVersion>
```

Comment:
Hmm.. why was this needed?

Reply:
Because I'm using a switch expression. It doesn't change execution-time requirements at all.